src/Nethermind/Nethermind.Evm/VirtualMachine.cs

```diff
@@ -1545,6 +1545,31 @@ static void UpdateCurrentState(EvmState state, int pc, long gas, int stackHead)
                         vmState.Memory.SaveByte(in memPosition, data);
                         if (_txTracer.IsTracingInstructions) _txTracer.ReportMemoryChange((long)memPosition, data);
 
+                        break;
+                    }
+                case Instruction.MCOPY:
```

Comment:
btw do you know if we have Ethereum state tests for MCOPY or not yet?

Reply:
there is this [PR](https://github.com/ethereum/tests/pull/1229/files) that was shared today

Reply:
Can we run them? :)

Reply:
will try to do it asap